api/src/test/java/org/jboss/cdi/api/test/privileged/annotation/AnnotationLiteralTest.java

```diff
@@ -0,0 +1,55 @@
+/*
+ * JBoss, Home of Professional Open Source
+ * Copyright 2018, Red Hat, Inc., and individual contributors
+ * by the @authors tag. See the copyright.txt in the distribution for a
+ * full listing of individual contributors.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ * http://www.apache.org/licenses/LICENSE-2.0
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.jboss.cdi.api.test.privileged.annotation;
+
+import org.testng.Assert;
+import org.testng.annotations.Test;
+
+/**
+ *
+ * This class test usage of privileged bloc in {@link javax.enterprise.util.AnnotationLiteral}
+ * They launched with Java security Manager.
```

Comment:
Where do we run this test with a `SecurityManager` used?

Reply:
Ah, you pushed one commit before this PR... https://github.com/cdi-spec/cdi/commit/c9aedeaadc9828cf2b5b8c3efc37f733a8f7569a

Reply:
Yes I hesitated but I have several PR to send with SM activated so as this addition doesn't change anything to release and test I found more convenient to have this on the common branch.